README.md

```diff
@@ -3,10 +3,73 @@
 
 For more information please refer to the main [Apiary](https://github.com/ExpediaInc/apiary) project page.
 
+## Variables
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| alerting_email | Email to receive alerts | string | - | yes |
+| cpu | The number of cpu units to reserve for waggledance container | string | `1024` | no |
+| docker_image | waggledance docker image | string | - | yes |
```

Comment:
Similar to other PR, please ensure all these Descriptions start with an upper case. Also waggledance -> Waggle Dance throughout and similar capitalisations of product names etc.